benchmarks/util.py

```diff
@@ -41,7 +41,7 @@ def reset_rng_state(benchmark_experiment=None):
   torch.manual_seed(1337)
   random.seed(1337)
   np.random.seed(1337)
-  if benchmark_experiment and benchmark_experiment.xla:
+  if benchmark_experiment is not None and benchmark_experiment.xla is not None:
```

Comment:
If benchmark_experiment and benchmark_experiment.xla are empty, would you do the same thing? If so, you can keep it as is as `if benchmark_experiment and benchmark_experiment.xla`.

Reply:
This is from a base PR. Sorry, I'm still learning how to use github reviews properly.

I'm still curious now and will shamelessly exploit this opportunity to learn something :) Why would if benchmark_experiment and benchmark_experiment.xla be more pythonic?

I don't have a strong opinion on this but my thinking is this: benchmark_experiment is of type Optional[BenchmarkExperiment] so I would like to easily distinguish between the None case and the BenchmarkExperiment case.